conftest.py

```diff
@@ -78,12 +79,31 @@ def appliance_police():
         port_results = {pn: net_check(pp) for pn, pp in ports.items()}
```

Comment:
Should we perhaps make it so that we retry the check after we process the exceptions? Even though the check is contained within. What it may be nice to do is have a port_fail_function for each port? That way if we incorporate more checks, we can have fail funcs for them too? Maybe I'm over thinking this though.